src/sentry/utils/meta.py

```diff
@@ -6,9 +6,9 @@
 
 class Remark(TypedDict):
     rule_id: str
-    ty: RemarkType
-    range_start: Optional[int]
-    range_end: Optional[int]
+    type: RemarkType
+    range_start: Optional[bytes]
+    range_end: Optional[bytes]
```

Comment:
The ranges here are byte offsets, but not bytes.

Reply:
Ah I completely misunderstood you earlier comment, sorry. So int is fine? Or is there a stronger type hint we can use.

Reply:
I don't think we have a stronger type hint, and it's also an overkill. We could add a comment on either `Remark` or `add_remark`.